File: app/src/main/java/ru/systempla/weatherapp/mvp/view/list/ForecastItemView.java
package ru.systempla.weatherapp.mvp.view.list;

public interface ForecastItemView {

    int getPos();
    void setDateTime(long dt);
    void setTemperature(float temp);
    void setWeatherDescription(String description);
    void setWeatherIcon(int actualId, long sunrise, long sunset);
}

File: app/src/main/java/ru/systempla/weatherapp/mvp/view/list/ForecastItemView.kt
package ru.systempla.weatherapp.mvp.view.list

interface ForecastItemView {
    val pos: Int

    fun setDateTime(dt: Long)
    fun setTemperature(temp: Float)
    fun setWeatherDescription(description: String)
    fun setWeatherIcon(actualId: Int, icon: String)
}